### ISSUE TITLE
v3: Cannot run an in place migration from an Umbraco 8 site

### ISSUE THREAD

**Issue by rickbutterfield** [skybrud/Skybrud.Umbraco.Redirects#111]
I'm trying to migrate an Umbraco 8 site to v9, using a copy of the existing database, and the startup is failing on the migration.

![image](https://user-images.githubusercontent.com/200450/138116481-16c6a68c-c9b6-4649-ad28-51d9ac6470d9.png)

```
{"@t":"2021-10-20T10:13:00.6547452Z","@mt":"Unhandled exception in AppDomain (terminating).","@l":"Error","@x":"System.InvalidOperationException: The migration plan does not support migrating from state \"2.0.5\".\r\n   at Umbraco.Cms.Infrastructure.Migrations.MigrationPlan.ThrowOnUnknownInitialState(String state)\r\n   at Umbraco.Cms.Infrastructure.Migrations.MigrationPlanExecutor.Execute(MigrationPlan plan, String fromState)\r\n   at Umbraco.Cms.Infrastructure.Migrations.Upgrade.Upgrader.Execute(IMigrationPlanExecutor migrationPlanExecutor, IScopeProvider scopeProvider, IKeyValueService keyValueService)\r\n   at Skybrud.Umbraco.Redirects.Components.MigrationComponent.Initialize()\r\n   at Umbraco.Cms.Core.Composing.ComponentCollection.Initialize()\r\n   at Umbraco.Cms.Infrastructure.Runtime.CoreRuntime.StartAsync(CancellationToken cancellationToken)\r\n   at Microsoft.Extensions.Hosting.Internal.Host.StartAsync(CancellationToken cancellationToken)\r\n   at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.RunAsync(IHost host, CancellationToken token)\r\n   at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.RunAsync(IHost host, CancellationToken token)\r\n   at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.Run(IHost host)\r\n   at Valley.Church.Web.Program.Main(String[] args) in C:\\Development\\Valley.Church\\Valley.Church.Web\\Program.cs:line 11","SourceContext":"Umbraco.Cms.Infrastructure.Runtime.CoreRuntime","ProcessId":25304,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":1,"AppDomainAppId":"35e6f600075658c8429a533dbee68fa787b914fe","MachineName":"UKL094","Log4NetLevel":"ERROR"}
```

Is it possible to go from v2.0.9 on Umbraco 8.17 to v3.0.0-alpha005 on Umbraco 9.0.1?

**Comment by rickbutterfield** [comment]
Fixed in #112

**Comment by abjerner** [comment]
Closing the issue as #112 has now been merged. Thanks for the help @rickbutterfield 👍
